### PR TITLE
Mohammad murtuza

### LINKED ISSUE #14: Navigate Board with Arrow Keys
Add the ability to change selected cell on Sudoku grid using arrow keys

### PR DESCRIPTION
**Pull request by mohammadmurtuza**
close #14

### CODE REVIEW

File: frontend/src/components/Board.jsx [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
   const [sudokuGrid, setSudokuGrid] = useState(Array.from({ length: 9 }, () => Array(9).fill("")));
-  const [selectedCell, setSelectedCell] = useState({ row: null, col: null });
-
-  const handleCellChange = (row, col, e) => {
-    const value = e.target.value;
-    if (/^[1-9]$/.test(value) || value === "") {
-      const newGrid = [...sudokuGrid];
-      newGrid[row][col] = e.target.value;
-      setSudokuGrid(newGrid);
-    }
```
Comment (derekjtong) [inline frontend/src/components/Board.jsx:15]: Character input handling was deleted, please add this back in

**Review by derekjtong — CHANGES_REQUESTED** [review]
Some main changes were overwritten

File: frontend/src/components/Board.jsx (hunk 1 above)
Comment (mohammadmurtuza) [reply frontend/src/components/Board.jsx:15]: fixed

File: frontend/src/components/Board9x9.jsx [hunk 2]
```diff
@@ -1,18 +1,15 @@
-import { useState } from "react";
+import React, { useState, useEffect } from "react";
```
Comment (derekjtong) [inline frontend/src/components/Board9x9.jsx:1]: No need to import React here

File: frontend/src/components/Board4x4.jsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from "react";
```
Comment (derekjtong) [inline frontend/src/components/Board4x4.jsx:1]: No need to import React here

**Review by derekjtong — CHANGES_REQUESTED** [review]

**Review by derekjtong — APPROVED** [review]

**Comment by derekjtong** [comment]
Close #15

**Pull request merged**